AChecker_Arano (updated)

kielarano opened the pull request.

Commit:
Updated AChecker

kevzter18 reviewed (changes requested):
1.) You need to merge into Development and not Master Branch
2.) Mukhang ganda naman, Bordz
3.) Goodjob

ghost commented:
Thank you par. Nacorrect ko na to development.

Commit:
Fixed AChecker

Corrected the part where A is getting removed and life is decreased even when it is correct.

Commit:
AChecker Final

Corrected mistakes
